attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/validator/StringLengthValidatorImpl.java

```diff
@@ -0,0 +1,63 @@
+package org.hypertrace.core.attribute.service.validator;
+
+import static com.google.protobuf.Descriptors.FieldDescriptor.JavaType.STRING;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.protobuf.Descriptors.Descriptor;
+import com.google.protobuf.Descriptors.FieldDescriptor;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.Message;
+import com.google.protobuf.util.JsonFormat;
+import com.google.protobuf.util.JsonFormat.Printer;
+import io.grpc.Status;
+import java.util.List;
+import java.util.function.Supplier;
+
+public class StringLengthValidatorImpl implements StringLengthValidator {
+  private static final int MAX_STRING_LENGTH = 1000;
+  private static final ObjectMapper MAPPER = new ObjectMapper();
+  private static final Printer PRINTER = JsonFormat.printer();
+
+  @Override
+  public void validate(final Message message) {
+    final Descriptor descriptor = message.getDescriptorForType();
+    try {
+      final JsonNode node = MAPPER.readTree(PRINTER.print(message));
```

Comment:
Why are we converting a proto message to string then back to a generic json object? If we want this generic (which is questionable, given we only have one message here right?) then we should be working with at the proto level rather than json.

Reply:
> If we want this generic (which is questionable, given we only have one message here right?)

Basically, we are trying to validate all the string values that are a part of the proto (even if they are nested inside other messages, one-ofs, or lists). Any string value introduced in the future would be automatically validated.

> we should be working with at the proto level rather than json

Right. Updated the implementation.